packages/shared/lib/src/sdk/mod.rs

```diff
@@ -235,6 +236,17 @@ impl Sdk {
                 .map_err(JsError::from)?;
                 transfer.0
             }
+            TxType::IBCTransfer => {
+                let args = tx::ibc_transfer_tx_args(tx_msg, None)?;
+                let transfer = namada::ledger::tx::build_ibc_transfer(
+                    &self.client,
+                    &mut self.wallet,
+                    args.clone(),
+                )
+                .await
+                .map_err(JsError::from)?;
```

Comment:
In many cases you can just do `await?` without map_err. The reason is that JsError implements From\<std::error::Error\>
```rs
    impl<E> From<E> for JsError
    where
        E: std::error::Error,
    {
        fn from(error: E) -> Self {
            JsError::new(&error.to_string())
        }
    }
```